torch/csrc/jit/mobile/backport_manager.cpp

```diff
@@ -212,6 +213,35 @@ bool backport_v5_to_v4(
   return true;
 }
 
+bool backport_v6_to_v5(
```

Comment:
OOC, is there a finite window for keeping this backport functionality?

Reply:
It's from current to 4.